docs/manifest/functionfile.md

```diff
@@ -58,9 +58,12 @@ function trackMessage (event) {
     // save this message
     event.completed();
 }
+
+// Register the function with Office.
+Office.actions.associate("trackMessage", trackMessage);
 ```
 
-The following code shows how to implement the function used by `FunctionName`.
+You can also define and register the function specified by the **FunctionName** element in a separate JavaScript file that is loaded by the HTML file. The following is an example of such a file.
 
 ```js
 // The initialize function must be run each time a new page is loaded.
```

Comment:
Can we make this initialize match the previous one so it is not in an anonymous function?